blueprints/showbie-tooling/files/.template-lintrc.js

```diff
@@ -1,7 +1,24 @@
 module.exports = {
-  extends: 'recommended',
-  ignore: [],
+  // extends: 'recommended',
+  ignore: [
+    'showbie/templates/noscript',
```

Comment:
We should either omit the `ignore` key from blueprint or just limit it to `ignore: ['dummy/**/*']` for addons. Again, this is something that's very application specific.